nulogy-fitter-happier.gemspec

```diff
@@ -25,12 +25,12 @@ Gem::Specification.new do |s|
   s.test_files   = Dir["spec/**/*"]
   s.require_path = 'lib'
 
-  s.add_dependency 'actionpack', '>= 6.0', '< 8.0'
-  s.add_dependency 'activerecord', '>= 6.0', '< 8.0'
+  s.add_dependency 'actionpack', '>= 7.2', '< 8.0'
```

Comment:
I would prefer that we specify `'>= 7.0'` as we are still using this gem with earlier versions of `Rails` (in particular `DQI` and `PackManager`).